models.py

```diff
@@ -0,0 +1,153 @@
+from datetime import datetime
+from werkzeug.security import generate_password_hash, check_password_hash
```

Comment:
Check what kind of hash it is using `generate_password_hash` function. This hash should be a slow one specifically designed for this kind of things. These hash functions should be "designed to be computationally intensive". See: `bcrypt`, `scrypt`, `blowfish`, `Argon2`, etc.
https://en.wikipedia.org/wiki/Key_derivation_function#Password_hashing
It is your responsibility as a developer to check this. (In real life, almost nobody does... which is very sad)